install/installer/pkg/components/auth/config.go

```diff
@@ -21,17 +22,40 @@ type Config struct {
 
 type SessionConfig struct {
 	// How long shoud the session be valid for?
-	LifetimeSeconds int64  `json:"lifetimeSeconds"`
-	Issuer          string `json:"issuer"`
+	LifetimeSeconds int64        `json:"lifetimeSeconds"`
+	Issuer          string       `json:"issuer"`
+	Cookie          CookieConfig `json:"cookie"`
+}
+
+type CookieConfig struct {
+	Name     string `json:"name"`
+	MaxAge   int64  `json:"maxAge"`
+	SameSite string `json:"sameSite"`
+	Secure   bool   `json:"secure"`
+	HTTPOnly bool   `json:"httpOnly"`
 }
 
 func GetConfig(ctx *common.RenderContext) ([]corev1.Volume, []corev1.VolumeMount, Config) {
 	volumes, mounts, pki := getPKI()
+	lifetime := int64((7 * 24 * time.Hour).Seconds())
 	return volumes, mounts, Config{
 		PKI: pki,
 		Session: SessionConfig{
-			LifetimeSeconds: int64((7 * 24 * time.Hour).Seconds()),
+			LifetimeSeconds: lifetime,
 			Issuer:          fmt.Sprintf("https://%s", ctx.Config.Domain),
+			Cookie: CookieConfig{
```

Comment:
This becomes the canonical place to set the JWT cookie properties across both components.

Reply:
I understand why we need to pull this config out. But honestly, it's nothing we _should_ configure as we need.

Would love to see a "CAUTION" comment here. Incl. some comments on the config settings, esp. highlighting the security implications. :orange_heart:

Reply:
Will do. It's not exposed to the installer, just a way to share the config between the components, which should limit the exposure

Reply:
Added